cirq/sim/mux.py

```diff
@@ -23,11 +23,19 @@
 
 from cirq import circuits, protocols, study, devices, ops, value
 from cirq.sim import sparse_simulator, density_matrix_simulator
+from cirq.sim.clifford import clifford_simulator
 
 if TYPE_CHECKING:
     import cirq
 
 
+def _is_clifford_circuit(program: 'cirq.Circuit') -> bool:
+    return all(
+        op.gate in clifford_simulator.CliffordSimulator.get_supported_gates()
```

Comment:
Don't recompute the supported gates list for every single operation.

Reply:
Done.